Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -63,6 +63,8 @@ protected void loadBanner(Context context, CustomEventBannerListener customEvent
         mCustomEventBannerListener = customEventBannerListener;
         pendingRequestBanner.set(true);
 
+        setAutomaticImpressionAndClickTracking(false);
```

Comment:
should this also be set for interstitial , I know that this came as suggestion from MoPUb for Banner, but might be relevant for Interstitial as well

Reply:
ok, fixed.